pkg/internal/apis/acme/install/pruning_test.go

```diff
@@ -17,20 +17,59 @@ limitations under the License.
 package install
 
 import (
-	"fmt"
+	"bytes"
+	"io/ioutil"
 	"testing"
 
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+	apiextensionsinstall "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/install"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	jsonserializer "k8s.io/apimachinery/pkg/runtime/serializer/json"
+	"k8s.io/apimachinery/pkg/runtime/serializer/versioning"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+
 	crdfuzz "github.com/munnerz/crd-schema-fuzz"
 
 	"github.com/jetstack/cert-manager/pkg/api"
 	acmefuzzer "github.com/jetstack/cert-manager/pkg/internal/apis/acme/fuzzer"
 )
 
 func TestPruneTypes(t *testing.T) {
-	crdfuzz.SchemaFuzzTestForCRDWithPath(t, api.Scheme, crdPath("orders"), acmefuzzer.Funcs)
-	crdfuzz.SchemaFuzzTestForCRDWithPath(t, api.Scheme, crdPath("challenges"), acmefuzzer.Funcs)
+	crdfuzz.SchemaFuzzTestForInternalCRD(t, api.Scheme, getCRD(t, "orders.acme.cert-manager.io"), acmefuzzer.Funcs)
+	crdfuzz.SchemaFuzzTestForInternalCRD(t, api.Scheme, getCRD(t, "challenges.acme.cert-manager.io"), acmefuzzer.Funcs)
 }
 
-func crdPath(s string) string {
-	return fmt.Sprintf("../../../../../deploy/charts/cert-manager/crds/%s.yaml", s)
+func getCRD(t *testing.T, resourceName string) *apiextensions.CustomResourceDefinition {
+	internalScheme := runtime.NewScheme()
+	utilruntime.Must(metav1.AddMetaToScheme(internalScheme))
+	apiextensionsinstall.Install(internalScheme)
```

Comment:
nit: we could make `internalScheme` a global var and call `Install` in an `init()` function to avoid constructing it each time, but it doesn't really matter that much in test code 🤷‍♂